### DISCUSSION TITLE
Touch input at X,Y

### DISCUSSION THREAD

**Discussion by Tyler-Lopez** [patrykandpatrick/vico#845] [Questions] [answered]
### Question

Is it possible to receive a touch input callback at a given X,Y? That would be fantastic, if so! I'm not seeing an example in the showcase.

This library does support "scrubbing" through a line chart somehow, right?

### UI framework(s)

Jetpack Compose

**Comment by Tyler-Lopez** [comment]
Answering myself partially in that I see how `CartesianMarker` fits into this.

I have a question related to this, if preferred I can make a separate thread: is it possible to show the marker at all times, even without user input? For example, I might want to always have a point in my line chart selected. It would be weird for the UI to not always have a point selected.

**Reply by Gowsky** [reply, marked as answer]
Hello, @Tyler-Lopez.

> Is it possible to receive a touch input callback at a given X,Y?

For charts with `CartesianMarker`s, this can be achieved via `CartesianMarkerVisibilityListener`. As mentioned in previous discussions, we plan on adding a dedicated, `CartesianMarker`-independent API for this. In the meantime, if you need such callbacks but don't want to have a `CartesianMarker`, an easy workaround is to create a `CartesianMarker` that doesn't draw anything.

> is it possible to show the marker at all times, even without user input?

Yes, it is. Please see `persistentMarkers`.

**Reply by Tyler-Lopez** [reply]
Hi there, thanks for the response!

This aligns with what I had found since asking this question: where my approach was to use a `persistentMarker` and then use a transparent non-persistent marker (because `CartesianMarkerVisibilityListener` depends upon the non-persistent marker - I realize you are mentioning it will be independent int he future, which is exciting!).

The "Y" bit I had mentioned I think is not achieved with your suggestion, but I think I had duplicated that part of the question elsewhere so will reply in more depth there - and apologies for that.
